### DISCUSSION TITLE
NO DATA in my GRAFANA - Status 500 - Internal Serveur Error

### DISCUSSION THREAD

**Discussion by glacman** [teslamate-org/teslamate#3022] [Getting started]
If you have any idea about these errors to collect data in Grafana ? 

![Capture d’écran 2022-12-17 à 18 09 20](https://user-images.githubusercontent.com/101400869/208253474-8d8344af-ab95-4290-b2ce-c46771b406fb.png)
![Capture d’écran 2022-12-17 à 18 09 57](https://user-images.githubusercontent.com/101400869/208253480-f2c9e2da-e06e-4bdd-b225-a1378e7f05db.png)
![Capture d’écran 2022-12-17 à 18 11 40](https://user-images.githubusercontent.com/101400869/208253489-ea920b5e-ddc1-4644-9947-9a4011a3ca21.png)

**Comment by ssgittyup** [comment]
Hi, I just installed this onto my Synology NAS and Grafana is showing the same errors. I am unable to see any data.  The Teslamate page at port 4000 loads just fine and shows the current location of the car with the relevant information, but no data shows up in Grafana and I see the same Query data errors. Not sure what to check next.  Thanks

**Comment by ssgittyup** [comment]
@glacman check this out, this seemed to be the same problem that we are having, I checked my yaml file and sure enough I had a space in front of the password, I followed this post below and was able to get everything working. 

https://github.com/adriankumpf/teslamate/discussions/2773#discussioncomment-3191930
